### PR TITLE
[Teams] Add descriptions to network logs

### PR DESCRIPTION
**Pull request by abracchi-tw**
- Add descriptions to network logs
- Move selective logging to Activity logs
- Tutorials cleanup

### CODE REVIEW

File: products/cloudflare-one/src/content/analytics/gateway.md [hunk 1]
```diff
@@ -7,10 +7,10 @@ pcx-content-type: concept
 
 Use the Analytics - Gateway page to see the top allowed and blocked requests across all of your locations. You can filter the data by selecting a specific location and/or time.
 
-![Overview of Gateway Analytics](../static/documentation/logs/gateway-analytics-overview.png)
+* **Requests.** This chart shows an overview of the number of requests made by end users within the time period you specify. It shows a breakdown of requests based on the decision Gateway took — `Allowed` or `Blocked`.
 
-## Selective Logging
+* **Allowed requests.** This chart shows a breakdown of the 5 domains which received the highest number of `Allowed` requests. It also shows the 5 domains which received the lowest amount of `Allowed` requests. By clicking **View all** to the right of either section of the chart, you'll see a list of highest 100 and lowest 100 domains ranked by number of `Allowed` requests.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/analytics/gateway.md]:
```suggestion
* **Allowed requests.** This chart shows a breakdown of the five domains which received the highest number of `Allowed` requests. It also shows the five domains which received the lowest amount of `Allowed` requests. By clicking **View all** to the right of either section of the chart, you will see a list of highest 100 and lowest 100 domains ranked by number of `Allowed` requests.
```

File: products/cloudflare-one/src/content/analytics/gateway.md [hunk 2]
```diff
@@ -7,10 +7,10 @@ pcx-content-type: concept
...
 
-By default, Gateway logs all events, including DNS queries and HTTP requests that are allowed and not a risk. You can customize what type of events are logged by navigating to **Settings** > **Network** and indicating your preferences under **Gateway Selective Logging**.
+* **Top blocked requests.** This chart shows a breakdown of the 5 domains which received the highest number `Blocked` requests. It also shows the 5 domains which received the lowest number of `Blocked` requests. By clicking **View all** to the right of either section of the chart, you'll see a list of highest 100 and lowest 100 domains ranked by number of `Blocked` requests.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/analytics/gateway.md]:
```suggestion
* **Top blocked requests.** This chart shows a breakdown of the five domains which received the highest number of `Blocked` requests. It also shows the five domains which received the lowest number of `Blocked` requests. By clicking **View all** to the right of either section of the chart, you'll see a list of highest 100 and lowest 100 domains ranked by number of `Blocked` requests.
```

File: products/cloudflare-one/src/content/tutorials/share-new-site.md [hunk 3]
```diff
@@ -108,21 +108,19 @@ When you create the DNS record, any visitor will be able to view that new site.
 
 Before you build the rule, you'll need to follow [these instructions](/setup) to set up Cloudflare Access in your account.
 
-Once enabled, navigate to the `Applications` page in the Cloudflare for Teams dashboard. Click `Add an application`.
+1. Once enabled, navigate to the `Applications` page in the Cloudflare for Teams dashboard.
 
-![Applications Page](../static/secure-origin-connections/share-new-site/applications.png)
+1. Click `Add an application`.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/share-new-site.md]: Should this be bold instead of monospace?

**Review by marciocloudflare — APPROVED** [review]

File: products/cloudflare-one/src/content/analytics/logs/activity-log.md [hunk 4]
```diff
@@ -68,6 +75,29 @@ When a user creates a policy to isolate traffic, the initial request that trigge
...
 
+## Network logs
+
+### Explanation of the fields
+
+<TableWrap>
+
+| Field | Description |
+| ----- | ----------- |
+| **Source IP** | The IP address of the user making the request. |
```
Comment (Encore-Encore) [inline products/cloudflare-one/src/content/analytics/logs/activity-log.md]:
```suggestion
| **Source IP** | The IP address of the user making the packet. | 
```

**Comment by QuincyLassiter** [comment]
في الجمعة، 12 نوفمبر 2021 3:39 م abracchi-tw ***@***.***> كتب:

**Pull request merged**